src/Raven.Server/Documents/PeriodicExport/PeriodicExportRunner.cs

```diff
@@ -406,7 +406,7 @@ private void WriteStatus()
             using (_database.DocumentsStorage.ContextPool.AllocateOperationContext(out context))
             using (var tx = context.OpenWriteTransaction())
             {
-                _database.DocumentsStorage.SetDatabasePeriodicExportStatus(context, _status);
+                _database.ConfigurationStorage.PeriodicBackupStorage.SetDatabasePeriodicBackupStatus(context, _status);
```

Comment:
Notice that we are passing the env from configuration storage, not document storage.